Userland/Libraries/LibC/wchar.h

```diff
@@ -13,8 +13,13 @@
 
 __BEGIN_DECLS
 
+// Note: wint_t is unsigned on gcc, and signed on clang.
 #ifndef WEOF
-#    define WEOF (0xffffffffu)
+#    ifdef __clang__
+#        define WEOF (-1)
+#    else
+#        define WEOF (0xffffffffu)
+#    endif
```

Comment:
Note to self: Patch GCC/Clang to have the same `__WINT_TYPE__`, as otherwise the Clang port on GCC (or vice versa) might break subtly.